measure_extinction/extdata.py

```diff
@@ -1113,21 +1118,18 @@ def plot(
             if curtype in exclude:
                 continue
             # replace extinction values by NaNs for wavelength regions that need to be excluded from the plot
-            self.exts[curtype][self.npts[curtype] == 0] = np.nan
+            bdata = self.npts[curtype] == 0
+            if np.any(bdata):
```

Comment:
Not sure why this is needed

Reply:
I was having an issue at one point, but maybe that was a red herring.  I will revert to the more compact (and pythonic?) way.